### DISCUSSION TITLE
Why do the DBT images generated by the FPB algorithm become smaller and smaller as the number of layers accumulates?

### DISCUSSION THREAD

**Discussion by ouyoung19** [marcelo-vieira/DBT-Reconstruction#14] [Q&A]
T
![17241489776353](https://github.com/user-attachments/assets/0fe96bda-672f-48d9-87b5-4b98da0d1103)
he parameters used in the code are correct, then why does the outcome appear as shown in the figure?

**Comment by rodrigovimieiro** [comment]
Can you elaborate on your question? I did not understand it

**Reply by ouyoung19** [reply]
I reconstructed a tomographic image using the FBP algorithm, and one of the frames is shown in the image above. Since there are overlapping regions on the top and bottom of the image, I'm not sure if this is correct.

**Comment by rodrigovimieiro** [comment]
You probably have some dead pixels in the detector edge that are showing in the reconstruction. They might be saturated or something like that. You can crop them

**Reply by ouyoung19** [reply]
Sure, thank you for your help.
